header.yaml

```
@@ -0,0 +1,10 @@
+default-headers: 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)